### PR TITLE
Refactoring

### LINKED ISSUE #18: Advanced annotation testing
Implement tests for advanced annotation using different datasets based on HMDB database for annotation.

### LINKED ISSUE #19: Add code coverage to tests
To assess the code coverage obtained by our tests - which will help in identifying dead and redundant code and therefore refactoring - the tests should be extended using the [covr](https://github.com/r-lib/covr) package.

### PR DESCRIPTION
**Pull request by hechth**
This PR includes refactorings of multilevelannotationste2 and 5 without actual functional changes.

The tests have been slightly adapted to be more robust and uniform across different test runners such as `devtools::test()` and `tools::testInstalledPackage()`. This also allows running the test with code coverage using the `covr` package. Current values are visible [here](https://app.codecov.io/gh/hechth/recetox-xMSannotator).

Closes #18 
Closes #19

### CODE REVIEW

**Review by maximskorik — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -1,15 +1,37 @@
 Package: xMSannotator
```
Comment (martenson) [inline DESCRIPTION:1]: should we make this similar to recetox.aplcms here: https://github.com/RECETOX/recetox-aplcms/blob/master/DESCRIPTION#L1

File: R/get_chemscorev1.6.71.R [hunk 2]
```diff
@@ -1,1593 +1,955 @@
...
-                rel_adduct_data<-mchemicaldata[which(mchemicaldata$Adduct%in%as.character(adduct_weights[,1])),]
-                
-                rel_adduct_module<-gsub(rel_adduct_data$Module_RTclust,pattern="_[0-9]*",replacement="")
-                
-                module_rt_group<-gsub(mchemicaldata$Module_RTclust,pattern="_[0-9]*",replacement="")
-                
-                mchemicaldata<-mchemicaldata[which(module_rt_group%in%rel_adduct_module),]
-                
+#' @export
+do_something <- function(i,
```
Comment (martenson) [inline R/get_chemscorev1.6.71.R:2]: is this function's changed name an improvement?

**Review by martenson — APPROVED** [review]

File: R/get_chemscorev1.6.71.R (hunk 2 above)
Comment (hechth) [reply R/get_chemscorev1.6.71.R:2]: No it isn't, but I still don't really know what it does!

Comment (martenson) [reply R/get_chemscorev1.6.71.R:2]: Yeah, I went through it a little and then merged. It is confusing at the very least. 🤔

**Pull request merged**